R/lasso-deployment.R

```diff
@@ -35,27 +33,28 @@
 #' ## 1. Loading data and packages.
 #' ptm <- proc.time()
 #' library(healthcareai)
-#'
+#' 
 #' # setwd('C:/Yourscriptlocation/Useforwardslashes') # Uncomment if using csv
 #' 
 #' # Can delete this line in your work
 #' csvfile <- system.file("extdata", 
 #'                        "HCRDiabetesClinical.csv", 
 #'                        package = "healthcareai")
-#'
+#' 
 #' # Replace csvfile with 'path/file'
 #' df <- read.csv(file = csvfile, 
 #'                header = TRUE, 
 #'                na.strings = c("NULL", "NA", ""))
-#'
-#' head(df)
-#' str(df)
+#' 
+#' df$PatientID <- NULL # remove this column
+#' 
+#' 
+#' dfDeploy <- df[951:1000,]
```

Comment:
This line may be confusing to people who aren't too familiar with machine learning yet.  I suggest add a comment saying, "#save a dataframe for validation later on."